### PR TITLE
[IMP] estate: add computed fields and onchanges T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
In this PR I implement computed fields and onchanges for `estate.property` and `estate.property.offer`. The details are included in the commit message.

This is ready for review @deivislaya

Regards!

(I'll rebase over branch 16.0 once #7 is merged.)

### CODE REVIEW

File: estate/models/estate_property.py [hunk 1]
```diff
@@ -93,5 +94,36 @@ class Property(models.Model):
         help="Offerings for the property.",
     )
 
+    total_area = fields.Integer(
+        compute="_compute_total_area",
+        help="Total area of the property.",
+    )
+    best_price = fields.Float(
+        string="Best Offer",
+        default=0.0,
```
Comment (deivislaya) [inline estate/models/estate_property.py]: Please. Delete attribute

File: estate/models/estate_property.py [hunk 2]
```diff
@@ -93,5 +94,36 @@ class Property(models.Model):
...
+            record.best_price = max(offered_prices, default=0.0)
+
+    @api.onchange("garden")
+    def _onchange_garden(self):
+        if self.garden:
+            self.garden_area = 10
+            self.garden_orientation = "north"
+        else:
+            self.garden_area = 0
+            self.garden_orientation = ""
```
Comment (deivislaya) [inline estate/models/estate_property.py]: self.garden_orientation = False

**Comment by Suavesito-Olimpiada** [comment]
Ready for review, ping @deivislaya.

**Comment by Suavesito-Olimpiada** [comment]
Rebased.

**Pull request merged**